lib/atlas/dataset/derived_dataset.rb

```diff
@@ -4,6 +4,7 @@ class Dataset::DerivedDataset < Dataset
 
     attribute :base_dataset, String
     attribute :scaling,      Preset::Scaling
+    attribute :init,         Hash
```

Comment:
Better do `Hash[Symbol => Float]` as in [`Atlas::Preset`](https://github.com/quintel/atlas/blob/c58d46cfbc995ec361ab2bc1559fe13944528f18/lib/atlas/preset.rb#L18)